### PR TITLE
Update Readme to inform users of v2.

### PR DESCRIPTION
**Pull request by ojongerius**
After this goes in we can create create tag v1 on master.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,37 +4,55 @@ status](https://travis-ci.org/zorkian/go-datadog-api.svg)](https://travis-ci.org
...
-Hi!
+A Go wrapper for the Datadog API. Use this library if you need to interact with the Datadog system. You can post metrics
+with it if you want, but this library is probably mostly used for automating dashboards/alerting and retrieving data
+(events, etc).
 
-This is a Go wrapper for the Datadog API. You should use this library if you need to interact
-with the Datadog system. You can post metrics with it if you want, but this library is probably
-mostly used for automating dashboards/alerting and retrieving data (events, etc).
+The master branch contains the v1 version of this library. If you are going to use this library for the first time you
+should switch to the v2 branch. If you are a v2 user, please upgrade to the v2 branch. This version is currently soaking,
```
Comment (zorkian) [inline README.md]: `If you are a v2` -> `If you are a v1`

File: README.md [hunk 2]
```diff
@@ -4,37 +4,55 @@ status](https://travis-ci.org/zorkian/go-datadog-api.svg)](https://travis-ci.org
...
 The source API documentation is here: <http://docs.datadoghq.com/api/>
 
-
-## USAGE
-
-To use this project, include it in your code like:
-
-``` go
+## Installation
+### v1
```
Comment (zorkian) [inline README.md]: Maybe a note here like,

`### v1 (deprecated)`

?

File: README.md [hunk 3]
```diff
@@ -65,4 +82,4 @@ in your environment variables.
 
 Please see the LICENSE file for the included license information.
 
-Copyright 2013 by authors and contributors.
+Copyright 2017 by authors and contributors.
```
Comment (zorkian) [inline README.md]: I think technically it should be `2013-2017`, but I'm not sure it strictly matters.

File: README.md (hunk 2 above)
Comment (ojongerius) [inline README.md]: I like it, done.

File: README.md (hunk 1 above)
Comment (ojongerius) [inline README.md]: Thanks!

File: README.md (hunk 3 above)
Comment (ojongerius) [inline README.md]: Agreed.

**Pull request merged**